src/backgroundblurvideoframeprocessor/BackgroundBlurVideoFrameProcessorDelegate.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import BackgroundBlurVideoFrameProcessorObserver from './BackgroundBlurVideoFrameProcessorObserver';
+
+/** @internal */
+export default class BackgroundBlurVideoFrameProcessorDelegate {
```

Comment:
Can we add comments here for these classes for what they do?

Reply:
added comments